packages/babel-helper-builder-react-jsx/src/index.js

```diff
@@ -14,9 +14,12 @@ export default function(opts) {
   const visitor = {};
 
   visitor.JSXNamespacedName = function(path) {
-    throw path.buildCodeFrameError(
-      "Namespace tags are not supported. ReactJSX is not XML.",
-    );
+    if (opts.throwIfNamespace) {
+      throw path.buildCodeFrameError(
+        "Namespace tags are not supported by default. React's JSX doesn't support namespace tags." +
+          "You can turn on the 'throwIfNamespace' flag to bypass this warning.",
```

Comment:
Does the template literal not work here? looks like this is all one line, and you need a space `tags.You`

Reply:
@hzoo You are right. I'm missing a space, good catch. I've used template literals, but then I run into "line is too long' warning. But now I discovered that there is `line continuation (\)` so I'm going to rewrite this.